client/src/pages/WeeklyTable.js

```diff
@@ -74,12 +76,55 @@ const WeeklyTable = ({
 	refreshBooking,
 	maxDesksForDay,
 }) => {
-	const bookingsByRow = getBookingsByRow(bookings, rowsCount, week);
-	console.log({ bookingsByRow });
+	const [currentMonday, setCurrentMonday] = useState(new Date(2022, 0, 17));
+	let week = [];
+	const year = currentMonday.getFullYear();
+	const month = currentMonday.getMonth();
+	let date = currentMonday.getDate();
```

Comment:
I'm wondered why shouldn't use "let" either for month and year!

Reply:
good question